autofs-5.1.2 - fix release date in CHANGELOG

From: Ian Kent <raven@themaw.net>

Oops!

Also add heading for 5.1.3.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CHANGELOG b/CHANGELOG
index 39019a9..d31bb80 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,4 +1,8 @@
-15/06/2015 autofs-5.1.2
+xx/xx/2016 autofs-5.1.3
+=======================
+- fix release date in CHANGELOG.
+
+15/06/2016 autofs-5.1.2
 =======================
 - update libtirpc workaround for new soname.
 - revert fix libtirpc name clash.
